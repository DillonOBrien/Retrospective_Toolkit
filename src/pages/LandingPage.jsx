import React from 'react';
import DiceLanding from '../containers/DiceLanding';
import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
import Red from '@material-ui/core/colors/red';

const theme = createMuiTheme({
    palette: {
        primary: {
            main: Red[600]
        },
        secondary: {
            main: '#3b3535'
        }
    }
})

const LandingPage = () => {

    return (
        <ThemeProvider theme={theme}>
            <DiceLanding />
        </ThemeProvider>
    );
}

export default LandingPage;