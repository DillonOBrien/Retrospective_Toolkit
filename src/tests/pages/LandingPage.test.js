import React from "react";
import { shallow } from "enzyme";
import LandingPage from "../../pages/LandingPage";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<LandingPage />);
});

describe("Pages/LandingPage Testing", () => {
  // Snapshot for LandingPage
  test("SNAPSHOT: LandingPage", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
