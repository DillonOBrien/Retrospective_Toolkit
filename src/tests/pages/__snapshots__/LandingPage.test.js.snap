// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/LandingPage Testing SNAPSHOT: LandingPage 1`] = `<DiceLanding />`;
